README.Rmd

```diff
@@ -1,459 +1,475 @@
----

```

Comment:
I have no idea why there is such massive diff here. 

The best way to review the change is to look at the `README.md` file instead.